selfstate.go

```diff
@@ -6,9 +6,9 @@ import (
 	"time"
 )
 
-// CheckSelfStateMonitorSettings -
+// CheckSelfStateMonitorSettings - validate contact types
 func CheckSelfStateMonitorSettings() error {
-	if config.Notifier.SelfState.Enabled != "true" {
+	if ToBool(config.Notifier.SelfState.Enabled) {
 		return nil
```

Comment:
It seems to bug.
If self check enabled, we exit from function and do not check contact settings